tag/feature.py

```diff
@@ -12,6 +12,7 @@
 from tag.directive import Directive
 from tag.range import Range
 from tag.sequence import Sequence
+from tag.score import Score
```

Comment:
Should try to change all of these from `from tag.x import X` to `from tag import X`. It works elsewhere, it should work here.